libr/include/r_anal.h

```diff
@@ -2020,6 +2021,12 @@ typedef enum {
 	R_ANAL_CLASS_ERR_OTHER
 } RAnalClassErr;
 
+typedef struct r_anal_class_node_info_t {
+	char *title;
+	char *body;
+	ut64 offset;
+} RAnalClassNodeInfo;
```

Comment:
The purpose of having user data structure with these fields was so that isn't specific to single use case and can be reused by large portion of other graphs. This means that the structure needs to be declared in rgraph.h or soemwhere else in util, the structure name needs to be more generic. Similar changes apply to  rgraph->ragraph conversion function, but that one should probably be in ragraph.

Reply:
Yes, moving the function to agraph module and the generic data node into rgraph is much better.